<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmGPoint</class>
 <widget class="QWidget" name="FrmGPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblLon">
      <property name="text">
       <string>Lon:</string>
      </property>
      <property name="buddy">
       <cstring>editLon</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editLon">
      <property name="toolTip">
       <string>Enter Y coordinate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblLat">
      <property name="text">
       <string>Lat:</string>
      </property>
      <property name="buddy">
       <cstring>editLat</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editLat">
      <property name="toolTip">
       <string>Enter the X coordinate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblHeight">
      <property name="text">
       <string>Height:</string>
      </property>
      <property name="buddy">
       <cstring>editHeight</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editHeight">
      <property name="toolTip">
       <string>Enter Z coordinate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
